apps/src/aiComponentLibrary/suggestedPrompt/SuggestedPrompts.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import SuggestedPrompt, {SuggestedPromptProps} from './SuggestedPrompt';
+
+import moduleStyles from './suggested-prompt.module.scss';
+
+/**
+ * Renders clickable tags that can be customized with list of suggested prompts.
+ */
+
+interface SuggestedPromptsProps {
+  suggestedPrompts: Array<SuggestedPromptProps>;
+}
+
+const SuggestedPrompts: React.FC<SuggestedPromptsProps> = ({
+  suggestedPrompts,
+}) => {
+  return (
+    <div className={moduleStyles.prompts}>
+      {suggestedPrompts.map(prompt => (
+        <SuggestedPrompt
+          onClick={prompt.onClick}
+          label={prompt.label}
+          show={prompt.show}
+        />
+      ))}
+    </div>
+  );
```

Comment:
again, non-blocking, but just noting that since it's the single element returned we can drop the explicit block and return statement.